src/main/kotlin/id/walt/service/keri/KeriInceptionService.kt

```diff
@@ -0,0 +1,73 @@
+package id.walt.service.keri
+
+import id.walt.service.dto.KeriInceptionResponse
+import id.walt.service.keri.interfaces.KeriInceptionInterface
+import java.io.BufferedReader
+import java.io.IOException
+import java.io.InputStreamReader
+
+class KeriInceptionService: KeriInceptionInterface {
+
+    private val CONFIG_DIR = "config"
+    private val INCEPTION_FILE = "inception-config.json"
+    override fun inceptController(name: String, alias: String, passcode: String): KeriInceptionResponse {
+        var prefix = ""
+        val publicKeys = mutableListOf<String>()
+
+        val inceptCommand: List<String> = listOf(
+            "kli", "incept",
+            "--name" ,name,
+            "--alias", alias,
+            "--passcode", passcode,
+            "--file", "$CONFIG_DIR/keri/$INCEPTION_FILE"
+        )
+
+        // kli status --name waltid --alias waltid-alias | awk '/Identifier:/ {print $2}'
```

Comment:
If a wrong passcode is provided the request can go indefinitely. We should provide a TTL.